docs/docs/docs/getting-started/installation.md

```
@@ -196,7 +196,70 @@ Click on the image below to play the video.
+   cp envFiles/.env.devcontainer .env
+   ```
+1. Change the `.env` file's `API_POSTGRES_HOST` parameter to `localhost`.
+   ```ini
+   API_POSTGRES_HOST=localhost
+   ```
+1. Install the packages
+   ```bash
+   pnpm install
+   ```
+1. Intall the `devcontainers/cli` package
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)